### PR TITLE
[Merge Wednesday] Add blog for Drizzle

### PR DESCRIPTION
**Pull request by SferaDev**
## Summary

- Add blog for drizzle
- Update blog for prisma with same launch date and re-used images

### CODE REVIEW

File: drizzle-postgres-xata-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import { drizzle } from 'drizzle-orm/xata-http';
+import { getXataClient } from './xata'; // Generated client
+
+const xata = getXataClient();
+const db = drizzle(xata);
+```
+
+## Get started
+
+And that's it, it's that easy to connect Drizzle to your Xata database. This functionality is currently only available with the public beta of our Postges service.
```
Comment (SferaDev) [inline drizzle-postgres-xata-integration.mdx:106]:
```suggestion
And that's it, it's that easy to connect Drizzle to your Xata database. This functionality is currently only available with the public beta of our Postgres service.
```

**Review by alexfrancoeur — APPROVED** [review]
LGTM overall. I added one suggestion to the intro and two docs references. For the intro suggestion, I'm honestly not sure if it's a good one or not, but let's see what folks think.

File: drizzle-postgres-xata-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+tags: ['engineering', 'drizzle', 'announcements', 'launch-week']
+published: false
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-integrations/drizzle-integration.jpg
+  alt: 'Drizzle Integration'
+slug: drizzle-postgres-xata-integration
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-integrations/drizzle-integration.jpg
+---
+
+[Drizzle ORM](https://orm.drizzle.team/) is a very popular TypeScript ORM that provides type safe access to your database, automated migrations, and a custom data model definition.
```
[the remark this answers is not in the document]
Comment (CezzaineZ) [reply drizzle-postgres-xata-integration.mdx:16]: Ha! It does match the tone of their posts in general. We could remove from the blog given its not really our tone / style and might be seen badly in the community. But we could instead use this as a cheeky social post and tag them and that way entice a response / reshare as a result.

File: drizzle-postgres-xata-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
+---
+title: Using Drizzle with Xata's Postgres service
+description: 'Get started with Drizzle ORM and Xata'
+author: Alexis Rico
+authorEmail: alexis@xata.io
+date: 03-20-2024
+tags: ['engineering', 'drizzle', 'announcements', 'launch-week']
+published: false
```
Comment (SferaDev) [inline drizzle-postgres-xata-integration.mdx:8]:
```suggestion
published: true
```

File: prisma-integration.mdx [hunk 4]
```diff
@@ -3,19 +3,19 @@ title: Using Prisma with Xata's Postgres service
 description: 'Get started with Prisma ORM and Xata'
 author: Emily Morgan
 authorEmail: emily@xata.io
-date: 03-18-2024
+date: 03-20-2024
 tags: ['engineering', 'prisma', 'announcements', 'launch-week']
 published: false
```
Comment (SferaDev) [inline prisma-integration.mdx:8]:
```suggestion
published: true
```

**Pull request merged**